### ISSUE TITLE
Responsive/mobile layout misbehaving - grid layout

### ISSUE THREAD

**Issue by CosminAvasiloaei** [foundation/foundation-emails#1140]
**How can we reproduce this bug?**

Write out the HTML (or Inky code) that causes the issue.

`            <row class="item-row border-bottom">
                <columns small="7" large="6">
                    <p class="mb-0">2023 Home Adult Shirt</p>
                    <p class="mb-0 light-gray">Size: Large</p>
                    <p class="mb-0 light-gray">Color: White/Blue</p>
                </columns>
                <columns small="2" large="1">
                    <p class="text-center light-gray">x1</p>
                </columns>
                <columns small="3" large="2">
                    <p class="text-right ">&pound;20.00</p>
                </columns>
            </row>`
![image](https://user-images.githubusercontent.com/21079047/227500618-0a997536-b48d-4482-b72c-2a4756ba1047.png)

**What did you expect to happen?**
The specified number to reflect into the width of the content on mobile.

**What happened instead?**
Foundation seems to be overwriting their own width % with higher % as if it would have a higher number of columns specified for "small"; e.g. small-3 is given width of small-4

**Comment by CosminAvasiloaei** [comment]
Seemingly this is all done in a node_modules file of Foundation Emails so not something I can amend, nor do I want to start writing hundreds of lines to overwrite this sort of behaviour which occurs on multiple column widths

**Comment by viking1304** [comment]
Your screenshot shows that **small-3** = **25%**, which is the correct value.

The default grid has 12 rows, in which case the widths should be:

```
small-1 = 8.333333%
small-2 = 16.666666%
small-3 = 25%
small-4 = 33.333333%
...
small-6 = 50%
...
small-12 = 100%
```

If you consider this piece of code

```
<columns small="3"></columns>
<columns small="3"></columns>
<columns small="3"></columns>
<columns small="3"></columns>
```

it became very obvious that small-3 must be 25%

The same applies to this

```
<columns small="4"></columns>
<columns small="4"></columns>
<columns small="4"></columns>
```

small-4 should obviously has to be 33.333333%

In your case, with

```
<columns small="7"></columns>
<columns small="2"></columns>
<columns small="3"></columns>
```

you must get those values:

small-7 = 58.333333%
small-2 = 16.666666%
small-3 = 25%

**Comment by CosminAvasiloaei** [comment]
Yeah, I finally realized I was being silly and rushed to not trust into Foundation :D, although it seems to be acting silly in a different way.

Turns out what is actually happening is if you have line-break between your <columns> elements then it adds some whitespace to the page and messes up the layout. I have made sure I didnt have any whitespace in the code apart from the actual line-break, but it seems to only like it if the next <columns> element follows immediately after the previous closing <columns> tag.

![image](https://user-images.githubusercontent.com/21079047/227533776-56998977-bd9a-43d1-a981-5496adadf5df.png)
![image](https://user-images.githubusercontent.com/21079047/227533818-67c86f03-770e-49da-8906-57de28a56b83.png)
